tests/test_sourmash.py

```diff
@@ -1025,9 +1021,7 @@ def test_compare_choose_molecule_dna():
         testdata1 = utils.get_test_data('short.fa')
         testdata2 = utils.get_test_data('short2.fa')
         status, out, err = utils.runscript('sourmash',
-                                           ['compute', '-k', '30',
-                                            '--dna', '--protein',
-                                            testdata1, testdata2],
+                                           ['sketch', 'dna', '-p', 'k=30,num=500',testdata1, testdata2],
```

Comment:
This test now doesn't test what it was supposed to; it used to test choosing between multiple molecule types when running `sourmash compare` by using `--dna`, but now there's only one molecule type being generated, so no choice needs to be made. We'll need to take a deeper look at this; for now, can you revert it to using `sourmash compute`?

Reply:
Alright!